src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -27,6 +33,7 @@ public function __construct(
         private readonly string $charset = 'UTF-8',
         private readonly string|false $polyfillUrl = ImportMapManager::POLYFILL_URL,
         private readonly array $scriptAttributes = [],
+        private readonly ?RequestStack $requestStack = null,
```

Comment:
Injecting the request stack here feels a bit odd, but it's the best way I can think of to handle this.